go/p2p/txsync/protocol.go

```diff
@@ -1,8 +1,13 @@
 package txsync
```

Comment:
Unrelated to this PR, but I just noticed that `txsync` is defined in `go/p2p`, but it would probably make more sense for it to be moved under `go/worker/common/p2p` (similar to e.g. storage and key manager)?